Changelog.md

```diff
@@ -1,5 +1,9 @@
 ### 0.4.5 (unreleased)
 
+Features:
+
+ * Type checker: now more eagerly searches for a common type of tuple elements of mixed types
```

Comment:
I'm sorry to bug again, but actually this is not about tuples but about inline arrays.